TrailheadApex11/ExercisesPage/src/classes/Ex1SayingHello.cls

```diff
@@ -0,0 +1,18 @@
+/**
+ * Created by Kuba on 29.11.2017.
+ */
+
+public with sharing class Ex1SayingHello implements ExercisesLoader {
+    public String solveTheExercise(List<String> input) {
+        try {
+            String inputString = ExercisesController.isInputEmpty(input).get(0);
+            if (input == null) {
+                return 'You have to type text to be solved';
+            } else {
+                return 'What is your name? ' + inputString + '\nHello, ' + inputString + ', nice to meet you!';
+            }
+        } catch (Exception e) {
+            return 'You have to type text to be solved';
```

Comment:
To jest takie uciszenie wyjątku. Tu nie łap, wyjeb try/catcha i daj w kontrolerze. Z resztą to jest nawet DRY - teraz musisz robić to samo w każdej implementacji Zadania.

Nie wspominając już o tym, że masz ten string tyle razy zamiast w jakiejś klasie ze staticami.. ale tą uwagę CI sypię po całym kodzie i tak. A jak sobie teraz zażyczę, że chce mieć tą labelkę jednak inną, z wykrzyknikiem na końcy np to co, będziesz zapierdalał po całym kodzie i poprawiał? ;)
Wysłałem Ci gdzieś linka o tych magic tringach - o walkę z takimi rzeczami dokładnie chodzi. 

W SF są jeszcze na to Labelki, ale do tego dojdziemy w kolejnym kroku